package.json

```diff
@@ -1,110 +1,110 @@
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged",
-      "pre-push": "yarn validate"
-    }
-  },
-  "lint-staged": {
-    "src/**/*.{js,jsx,ts,tsx,json,md,scss,css}": [
-      "prettier --check"
-    ]
-  },
-  "devDependencies": {
-    "@types/react": "^16.9.33",
-    "@types/react-dom": "^16.9.6",
-    "@types/react-helmet": "^5.0.15",
-    "@types/react-responsive": "^8.0.2",
-    "@typescript-eslint/eslint-plugin": "^2.27.0",
-    "@typescript-eslint/parser": "^2.27.0",
-    "eslint": "^6.8.0",
-    "eslint-config-prettier": "^6.10.1",
-    "eslint-config-prettier-standard": "^3.0.1",
-    "eslint-config-standard": "^14.1.1",
-    "eslint-config-standard-react": "^9.2.0",
-    "eslint-loader": "^4.0.0",
-    "eslint-plugin-import": "^2.20.2",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.1.2",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-react": "^7.19.0",
-    "eslint-plugin-standard": "^4.0.1",
-    "gatsby-plugin-s3": "^0.3.3",
-    "gh-pages": "^2.2.0",
-    "husky": "^4.2.3",
-    "lint-staged": "^10.1.2",
-    "prettier": "^2.0.4",
-    "prettier-stylelint": "^0.4.2",
-    "rimraf": "^3.0.2",
-    "stylelint": "^13.3.0",
-    "stylelint-config-prettier": "^8.0.1",
-    "stylelint-config-recommended": "^3.0.0",
-    "stylelint-config-standard": "^20.0.0",
-    "typescript": "^3.8.3"
-  },
-  "repository": {
-    "type": "git",
-    "url": "github:vrk-kpa/suomifi-design-system"
-  },
-  "bugs": {
-    "url": "https://github.com/vrk-kpa/suomifi-design-system/issues"
-  }
-}
+{
+  "name": "suomifi-design-system",
+  "private": true,
+  "description": "Suomi.fi Design System",
+  "version": "0.1.2",
+  "author": "Jari Porola <jari.porola@gmail.com>",
+  "dependencies": {
+    "@csstools/normalize.css": "^10.1.0",
+    "@svgr/webpack": "^5.3.1",
+    "@wapps/gatsby-plugin-i18next": "^1.1.6",
+    "babel-plugin-styled-components": "^1.10.7",
+    "gatsby": "^2.20.14",
+    "gatsby-image": "^2.3.2",
+    "gatsby-plugin-manifest": "^2.3.3",
+    "gatsby-plugin-module-resolver": "^1.0.3",
+    "gatsby-plugin-offline": "^3.1.2",
+    "gatsby-plugin-react-helmet": "^3.2.1",
+    "gatsby-plugin-sharp": "^2.5.4",
+    "gatsby-plugin-styled-components": "^3.2.1",
+    "gatsby-plugin-typescript": "^2.3.1",
+    "gatsby-plugin-web-font-loader": "^1.0.4",
+    "gatsby-source-filesystem": "^2.2.2",
+    "gatsby-transformer-code": "^0.1.0",
+    "gatsby-transformer-sharp": "^2.4.4",
+    "i18next": "^19.4.0",
+    "prop-types": "^15.7.2",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-element-to-jsx-string": "^14.3.1",
+    "react-helmet": "^5.2.1",
+    "react-i18next": "^11.3.4",
+    "react-motion": "^0.5.2",
+    "react-responsive": "^8.0.3",
+    "react-syntax-highlighter": "^12.2.1",
+    "styled-components": "^5.1.0",
+    "suomifi-icons": "0.1.1",
+    "suomifi-ui-components": "^0.15.1"

```

Comment:
I think it should be without the hat, ^.

``` "suomifi-ui-components": "0.15.1"```

Reply:
Should now be fixed.